src/bun.js/test/jest.zig

```diff
@@ -855,13 +890,67 @@ pub const DescribeScope = struct {
     }
 
     pub fn shouldEvaluateScope(this: *const DescribeScope) bool {
-        if (this.tag == .skip or
-            this.tag == .todo) return false;
-        if (Jest.runner.?.only and this.tag == .only) return true;
-        if (this.parent != null) return this.parent.?.shouldEvaluateScope();
-        return true;
+        return switch (this.tag) {
+            .skip, .todo => false,
+            .only => Jest.runner.?.only,
```

Comment:
Is this correct? If we hit only but `Jest.runner.?.only` is false, do we want to fallthrough?